src/Config/AuthToken.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Config;
+
+/**
+ * Authenticator Configuration for Token Auth and HMAC Auth
+ */
+class AuthToken
```

Comment:
This class is a Config class for CI4.
Extend `CodeIgniter\Config\BaseConfig`.

Reply:
Oh sorry, I knew that.  Thank you for catching.